src/archivematicaCommon/etc/dbsettings

```diff
@@ -2,4 +2,4 @@
 user=archivematica
 password=demo
 host=localhost
-
+db=archivematica
```

Comment:
Do you mind setting this to the current default of `MCP`? I think we'd like to keep the existing default.